react/features/calendar-sync/components/MeetingList.native.js

```diff
@@ -100,10 +110,45 @@ class MeetingList extends Component<Props> {
                 disabled = { disabled }
                 onPress = { this._onPress }
                 onRefresh = { this._onRefresh }
+                renderListEmptyComponent = {
+                    this._getRenderListEmptyComponent
+                }
                 sections = { this._toDisplayableList() } />
         );
     }
 
+    _getRenderListEmptyComponent: () => Object
+
+    /**
+     * Returns a list empty component if a custom one has to be rendered instead
+     * of the default one in the {@link NavigateSectionList}.
+     *
+     * @private
+     * @returns {Component}
+     */
+    _getRenderListEmptyComponent() {
+        const { _calendarAccessStatus, t } = this.props;
+
+        if (_calendarAccessStatus === 'denied') {
```

Comment:
Is it worthwhile to make these statuses constants or would that be too much at this point?